src/core/validate.cpp

```diff
@@ -953,14 +1096,15 @@ namespace validate
             v1_equivalent_root["roles"]["snapshot"] = RoleKeys({ std::vector<std::string>(), 1 });
             v1_equivalent_root["roles"]["timestamp"] = RoleKeys({ std::vector<std::string>(), 1 });
 
-            auto allkeys = keys();
+            auto allkeys = all_keys();
             std::map<std::string, Key> v1_keys = allkeys.at("root").keys;
             v1_keys.insert(allkeys.at("key_mgr").keys.cbegin(), allkeys.at("key_mgr").keys.cend());
 
             v1_equivalent_root["keys"] = v1_keys;
             v1_equivalent_root["_type"] = "root";
             v1_equivalent_root["version"] = version();
             v1_equivalent_root["spec_version"] = "1.0.17";
+            v1_equivalent_root["expires"] = "some";
```

Comment:
some?

Reply:
good catch, it was a test. I will fix it